### ISSUE TITLE
support rasa demo files

### ISSUE THREAD

**Issue by harrywang** [gamma-lab/parrot#5]
we should add another example for rasa demo - load this as a predefined example as well. https://github.com/RasaHQ/rasa-demo

**Comment by leon0707** [comment]
domain: https://github.com/RasaHQ/rasa-demo/blob/master/domain.yml
nlu: https://github.com/RasaHQ/rasa-demo/blob/master/data/nlu/nlu.md
stories: https://github.com/RasaHQ/rasa-demo/tree/master/data/core

**Comment by leon0707** [comment]
entity, synonym, and lookup in the nlu are not supported at this moment.

**Comment by harrywang** [comment]
yes. as this as a guidance to add features. refer to rasa's annotation tool: https://github.com/RasaHQ/rasa-nlu-trainer (quite old without updates)
